tests.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-set -e
```

Comment:
We remove this because failed assertions now return an rc of 1 which would stop this test script with `set -e`.